camelot/view/controls/editors/customeditor.py

```diff
@@ -167,11 +168,13 @@ def action_menu_triggered(self, checked):
 
     def add_actions(self, action_routes, layout):
         for action_route in action_routes:
+            action = initial_naming_context.resolve(tuple(action_route))
```

Comment:
in de gui mag de action niet geretrieved worden

in principe heeft de gui geen toegang tot de initial naming context

de icon zou normaal moeten gezet worden door de actoin states